arangod/Cluster/Maintenance/DBServerAgencySync.cpp

```diff
@@ -283,6 +285,9 @@ DBServerAgencySyncResult DBServerAgencySync::execute() {
                 << "Error reporting to agency: _statusCode: " << r.errorCode()
                 << " message: " << r.errorMessage() << ". This can be ignored, since it will be retried automatically.";
           }
+          LOG_TOPIC("9b0b3", DEBUG, Logger::MAINTENANCE)
+              << "Loading current in ClusterInfo";
+          clusterInfo.loadCurrent();
```

Comment:
I think this was intentionally removed in `devel`, probably after your PR was started. We should avoid to use explicit `loadCurrent` calls and I think this one is unnecessary.